case 36 - understand mime sniffing

mgmschoene opened the pull request:
This testcase is to generally understand for which content-type headers the browsers perform MIME sniffing. 
The test is a little more complex, since the webdriver can get stucked, if the download windows pops. In this case the webdriver needs to be killed via system commands. Locally for Firefox this was successfully tested.
Also, since several combinations are tested, one test took around 100 seconds locally.

mgmschoene committed:
case 36 - understand mime sniffing

mgmschoene committed:
case 36 - resolve merge conflict

mgmschoene committed:
Merge branch 'master' into case36

sanhphanvan96 commented:
Hi @mgmschoene,
Thank you for the test case. 
Unfortunately, since the BrowserStack's remote web drivers can get stuck when the download pop-up appears or even silently download without the pop-up. This seems to be an issue of the browser driver. BrowserStack's remote web drivers cannot be killed by running system commands locally (from the test script). We investigated this test case but still have not found the solution yet. We will try to find another approach.

mgmschoene committed:
Case 36 - updated testcase to skip tests with Firefox. Result 8 indicates aborted Firefox test.

mgmschoene committed:
Case 36 - merge with case 35

mgmschoene committed:
Merge branch 'case36' of github.com:canitrust/backend into case36

mgmschoene commented:
Hi @sanhphanvan96, 
I updated the PR. As agreed, we will skip the test for Firefox for now.

sanhphanvan96 committed:
Fixed testcase 36

- Edited a bit for the descriptions of testcase 35, 36
- Fixed testcae 36 to spawn remote web drivers

sanhphanvan96 reviewed (approved):
Thanks @mgmschoene , 
We can run it now! :punch: :punch: :punch:
